lib/fog/azurerm/docs/network.md

```diff
@@ -122,6 +122,42 @@ Get a single record of Subnet
      puts "#{subnet.name}"
 ```
 
+## Attach Network Security Group with Subnet
+
+Attach Network Security Group with Subnet
+
+```ruby
+     subnet = azure_network_service.attach_network_security_group('/subscriptions/<Subscription Id>/resourceGroups/<Resource Group Name>/providers/Microsoft.Network/networkSecurityGroups/<Network Security Group Name>')
+     puts "#{subnet.network_security_group_id}"
```

Comment:
We are doing this every where in documentation to show that the updated value of object. This means when user attach nsg then he can print nsg_id on screen to see the updated result.